.travis.yml

```diff
@@ -75,12 +92,16 @@ matrix:
     - os: osx
       osx_image: xcode8.3
 
+    - os: osx
+      osx_image: xcode8.3
+      env: BUILD_DEPENDENCY_FROM_SOURCE=ON
+
 before_install:
   # ISSUE: we may want to use '/scripts/install-deps-ubuntu.sh' in the future.
   # At this moment, travis-ci supports up to ubuntu 14.04 and this environment
   # does not recognize sudo apt-get install libglfw3-dev
```

Comment:
Now we can remove this comment?

Reply:
I am still debugging. And yes I want to get rid of it. I want to use util/scripts/install-deps-ubuntu.sh.